### PR TITLE
Trinity: fix hardcoded paths

### PR DESCRIPTION
**Pull request by abretaud**
Hi,
Some sed stuff to replace hard coded paths.
I still have a problem with trimmomatic: I need to know the path to some files installed in $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/adapters/
How can I do it in conda?

### CODE REVIEW

**Comment by bgruening** [comment]
Just a thought, can we symlink the adapter to some fixed path in the trimmomatic recipe and specify here just `$PREFIX/trimmomatic/fixed_path`

**Comment by abretaud** [comment]
hum yes, why not
maybe I can create a link like this;
ln -s $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM $PREFIX/share/$PKG_NAME

**Comment by abretaud** [comment]
I think it should be ~ok, waiting for travis to finish its tests

**Comment by bgruening** [comment]
@abretaud the following packages are missing from the OS-X channel, but the linux builds are ok! Awesome!
```
  - jellyfish
  - collectl
  - fastool
  - parafly
  - slclust
```

**Comment by abretaud** [comment]
Hum, why these packages are not on the osx channel? Is it something I can change?

**Comment by bgruening** [comment]
@abretaud oh yeah, please add them to the osx-whitelist in your root directory.

**Comment by abretaud** [comment]
Oops, there is a compilation error for trinity on osx
I don't have any mac to try to fix it...

**Comment by bgruening** [comment]
The joy of OS-X. I will have a look later... sorry.

**Comment by abretaud** [comment]
I uncommented the make step for trinity: it is needed otherwise inchworm & co binaries are not compiled. But I'm afraid it won't compile as is...
I have to leave now, I can look into it after my holidays

**Comment by abretaud** [comment]
Back to business!
Tests pass for linux, but still fail on osx: do you have a clue why, or should we just skip osx test maybe?

**Comment by bgruening** [comment]
Seems to be bigger problems in other packages and also missing dependencies in the OS-X channel. I would say skip for the time being until we have get jellyfish and Co to build on OS-X.

Thanks and welcome back!

**Comment by abretaud** [comment]
Ok, all tests are green now :)

File: recipes/jellyfish/1.1.11/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,9 @@ source:
 requirements:
   build:
     - yaggo >=1.5.8
+    - gcc
```
Comment (bgruening) [inline recipes/jellyfish/1.1.11/meta.yaml]: Here you need to increase the build number as far as I can see.

**Comment by abretaud** [comment]
Done!

**Comment by bgruening** [comment]
Awesome thanks!

**Pull request merged**